public/bundles/tetranzselect2entity/js/select2entity.js

```diff
@@ -0,0 +1,116 @@
+                                    cache[key] = {
+                                        data: data,
+                                        time: cacheTimeout ? Date.now() + cacheTimeout : null
+                                    };
+                                    success(data);
+                                });
+                            } else {
+                                // return cached data with no ajax request
+                                success(cache[key].data);
+                            }
+                        } else {
+                            // no caching enabled. just do the ajax request
+                            if (request) {
+                                request.abort();
+                            }
+                            request = $.ajax(params).fail(failure).done(success).always(function () {
+                                request = undefined;
+                            });
+                        }
+                    },
+                    data: function (params) {
+                        var ret = {
+                            'q': params.term,
+                            'field_name': $s2.data('name')
+                        };
+
+                        var reqParams = $s2.data('req_params');
+                        if (reqParams) {
+                            $.each(reqParams, function (key, value) {
+                                ret[key] = $('*[name="'+value+'"]').val()
+                            });
+                        }
+
+                        // only send the 'page' parameter if scrolling is enabled
+                        if (scroll) {
+                            ret['page'] = params.page || 1;
+                        }
+
+                        return ret;
+                    },
+                    processResults: function (data, params) {
+                        var results, more = false, response = {};
+                        params.page = params.page || 1;
+
+                        if ($.isArray(data)) {
+                            results = data;
+                        } else if (typeof data == 'object') {
+                            // assume remote result was proper object
+                            results = data.results;
+                            more = data.more;
+                        } else {
+                            // failsafe
+                            results = [];
+                        }
+
+                        if (scroll) {
+                            response.pagination = {more: more};
+                        }
+                        response.results = results;
+
+                        return response;
+                    }
+                }
+            }, options || {}));
+        });
+        return this;
+    };
+})( jQuery );
+
+(function( $ ) {
+    $(document).ready(function () {
+        $('.select2entity[data-autostart="true"]').select2entity();
+    });
+})( jQuery );
```

Comment:
I removed this file in a different Pull Request because I thought it was not needed (since it is not linked by any HTML file). Did you add it by accident or is this file really needed for the feature?

Reply:
What happened was that at first we did not use CDN, we started using it and forgot to remove the file :smile:

Reply:
Oh, great 😋 That's what I thought